src/Sylius/Bundle/ReviewBundle/Updater/ReviewableAverageRatingUpdaterInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ReviewBundle\Updater;
+
+use Sylius\Component\Review\Model\ReviewInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface ReviewableAverageRatingUpdaterInterface
+{
+    /**
+     * @param ReviewInterface $review
+     */
+    public function update(ReviewInterface $review);
```

Comment:
You mean``ReviewInterface``? Why? You can configure plenty of review objects in application and each of them can be quite different, but if they implement ``ReviewInterface`` we're sure they have proper review properties that are required to use review system properly.